From 9c1b6c7c37d90f28ee2f3427cb85c45ecae266ca Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sun, 16 May 2021 00:34:28 +0200
Subject: [PATCH 1/1] doc: extension.rst missing in doc/usage/index.rst

'make htmldocs' results in a build warning

    checking consistency... doc/usage/extension.rst:
    WARNING: document isn't included in any toctree

Add the document to the index.

Fixes: 2f84e9cf06d3 ("cmd: add support for a new "extension" command")
Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 doc/usage/index.rst | 1 +
 1 file changed, 1 insertion(+)

diff --git a/doc/usage/index.rst b/doc/usage/index.rst
index 528b3c745e..1dad33c814 100644
--- a/doc/usage/index.rst
+++ b/doc/usage/index.rst
@@ -26,6 +26,7 @@ Shell commands
    conitrace
    echo
    exception
+   extension
    exit
    false
    fatinfo
-- 
2.30.2

